README.md

```diff
@@ -167,7 +182,8 @@ to generate HTML. Here are two options for importing the `@kitajs/html` package:
    ```
 
 2. **Use the register to add a global namespace**: Import the `register` to globally
-   register all necessary functions for convenience.
+   register all necessary functions for convenience. You don't need to do that if you are
+   using the `jsxImportSource` option in your `tsconfig.json`.
```

Comment:
Should we prefer /register import or `react-jsx` way of "globally" installing kitajs/html? I'm not sure if both would be good.

Reply:
I think that the best approach is to follow how preact do it https://preactjs.com/guide/v10/typescript/. They let the user choose how to configure Typescript between importing the h and Fragment functions into every file or using the "react-jsx" + "jsxImportSource" approach (automatic import done by the transpiler).

Side note here:
We should also consider adding some documentation on how to setup a project using babel or similar for non-typescript users.